webapp/src/components/JobsListing/index.tsx

```diff
@@ -0,0 +1,374 @@
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'flex-end',
+  },
+  blockIcon: {
+    cursor: 'pointer',
+    color: theme.palette.error.light,
+    '&:hover': {
+      color: theme.palette.error.main,
+    },
+  },
+  downloadIcon: {
+    cursor: 'pointer',
+    margin: theme.spacing(0.5),
+    color: theme.palette.primary.main,
+    '&:hover': {
+      color: theme.palette.secondary.main,
+    },
+  },
+  errorIcon: {
+    width: '18px',
+    height: 'auto',
+    cursor: 'pointer',
+    color: theme.palette.error.main,
+    '&:hover': {
+      color: theme.palette.error.dark,
+    },
+  },
+}));
+
+const mapState = (state: AppState) => ({
+  studies: state.study.studies,
+});
+
+const mapDispatch = ({
+  loadStudies: initStudies,
+  addWsListener: addListener,
+  removeWsListener: removeListener,
+  subscribeChannel: subscribe,
+  unsubscribeChannel: unsubscribe,
+});
+
+const connector = connect(mapState, mapDispatch);
+type ReduxProps = ConnectedProps<typeof connector>;
+type PropTypes = ReduxProps;
+
+const JobsListing = (props: PropTypes) => {
+  const { studies, loadStudies, addWsListener, removeWsListener, subscribeChannel, unsubscribeChannel } = props;
+  const classes = useStyles();
+  const [t] = useTranslation();
+  const { enqueueSnackbar } = useSnackbar();
+  const theme = useTheme();
+  const [jobs, setJobs] = useState<LaunchJob[]>([]);
+  const [downloads, setDownloads] = useState<FileDownload[]>([]);
+  const [tasks, setTasks] = useState<Array<TaskDTO>>([]);
+  const createNotif = useNotif();
+  const [loaded, setLoaded] = useState(false);
+  const [openConfirmationModal, setOpenConfirmationModal] = useState<string | undefined>();
+  const [messageModalOpen, setMessageModalOpen] = useState<string | undefined>();
+
+  const init = async () => {
+    setLoaded(false);
+    try {
+      if (studies.length === 0) {
+        const allStudies = await getStudies();
+        loadStudies(allStudies);
+      }
+      const allJobs = await getStudyJobs(undefined, false);
+      setJobs(allJobs);
+      const dlList = await getDownloadsList();
+      setDownloads(dlList);
+      const allTasks = await getAllMiscRunningTasks();
+      const dateThreshold = moment().subtract(1, 'm');
+      setTasks(allTasks.filter((task) => !task.completion_date_utc || moment.utc(task.completion_date_utc).isAfter(dateThreshold)));
+    } catch (e) {
+      logError('woops', e);
+      enqueueErrorSnackbar(createNotif, 'ça marche pas', e as AxiosError);
+    } finally {
+      setLoaded(true);
+    }
+  };
+
+  const renderStatus = (job: LaunchJob) => {
+    let color = theme.palette.grey[400];
+    if (job.status === 'success') {
+      color = theme.palette.success.main;
+    } else if (job.status === 'failed') {
+      color = theme.palette.error.main;
+    } else if (job.status === 'running') {
+      color = theme.palette.warning.main;
+    }
+    return (<Box className={classes.dot} style={{ backgroundColor: color }} />);
```

Comment:
You can use a material ui dot icon.
Ex: FiberManualRecordIcon in '@material-ui/icons/FiberManualRecord';